src/components/timeout-dialog/timeout-dialog.yaml

```diff
@@ -30,3 +30,10 @@ examples:
     messageSuffix : "there is a suffix for the message"
     keepAliveButtonText: "keepAliveButtonText"
     signOutButtonText: "signOutButtonText"
+  visualRegressionTests:
+    backstopScenarioOptions:
+      delay: 10000
```

Comment:
Found that for this test nothing was displaying, it's happening for other timeouts too but I just updated this one because it's one that shows all the features of a timeout dialog so it's enough to get us coverage for most changes to the timeout dialog with other examples being mostly in welsh or around the timeout itself